src/components/OAuthLoginButton.tsx

```diff
@@ -24,12 +36,62 @@ export const OAuthLoginButton: FC<OAuthLoginButtonParams> = ({
   }, [login, onSuccess, onFail]);
 
   return (
-    <TouchableOpacity
-      testID={tID('oauth-login-button')}
-      {...touchableOpacityProps}
-      onPress={_login}
-    >
-      {children}
-    </TouchableOpacity>
+    <View style={styles.button}>
+      <TouchableOpacity
+        testID={tID('oauth-login-button')}
+        {...touchableOpacityProps}
+        onPress={_login}
+        style={styles.touchableOpacity}
+      >
+        <View style={styles.content}>
+          <Text style={styles.label}>{label}</Text>
+        </View>
+      </TouchableOpacity>
+    </View>
   );
 };
+
+const defaultStyles = (theme: Theme) => {
+  const button: ViewStyle = {
+    flexDirection: 'row',
+  };
+  const touchableOpacity: ViewStyle = {
+    alignItems: 'center',
+    backgroundColor: theme.colors.secondary,
+    borderRadius: 5 * theme.roundness,
+
+    minWidth: 64,
+  };
+
+  const content: ViewStyle = {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+  };
+
+  const label: TextStyle = {
+    color: theme.colors.onPrimary,
+    marginVertical: 10,
+    marginHorizontal: 24,
+
+    fontFamily: Platform.select({
+      web: 'Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif',
+      ios: 'System',
+      default: 'sans-serif-medium',
+    }),
+
+    fontWeight: '500',
+    letterSpacing: 0.1,
+    lineHeight: 20,
+    fontSize: 14,
+  };
+
+  return { button, touchableOpacity, content, label };
+};
+
+declare module './BrandConfigProvider/types' {
```

Comment:
I wonder if we could use TypeScript paths here to make this a bit easier. i.e. `declare module '@style' {`

Reply:
I spent some time trying to figure this out, but it sound like you already know the way!  Awesome, will update in next PR